### PR TITLE
Issue-82 ChangeSet Versioning

### PR DESCRIPTION
**Pull request by SaschaLucius**
Introducing the possibility to apply changeSets depending on your database schema version.

@dieppa please have a look at my first approach of the change.
Let me know if there is something missing.
I have not looked at the spring code, if there also would be something to change.

### CODE REVIEW

**Comment by dieppa** [comment]
Thanks, we'll have a look

File: README.md [hunk 1]
```diff
@@ -214,6 +214,8 @@ a date etc.
 `runAlways` - _[optional, default: false]_ changeset will always be executed but only first execution event will be 
 stored in dbchangelog collection
 
+`version` - _[optional, default: "0"]_ defines a version on which this changeset is relate to. E.g. "0.1" means, this changeset should be applied to schema version 0.1 of your MongoDB.
+
```
Comment (dieppa) [inline README.md:219]: We should add a real scenario where this can be needed. I don't think users will be able to see where to apply this feature.

**Review by dieppa — COMMENTED** [review]
I think is fine. Simple but potentially useful feature for some scenario. I just made a comment about providing a bit more context on where this can be applied. You can provide your own experience without adding any private information ;)

Thanks

**Comment by SaschaLucius** [comment]
@dieppa documentation added.
Does this match your expectation?

**Comment by dieppa** [comment]
Will have a look . I will release it in the beta this evening or tomorrow. 

thanks

**Comment by SaschaLucius** [comment]
Thanks,
feel free to merge it and let me know if something is missing

**Review by dieppa — DISMISSED** [review]

**Comment by dieppa** [comment]
yeah, actually one last thing. Can you reflect this in the changelog.md? 

Thanks

**Comment by SaschaLucius** [comment]
added to 3.0.1

**Review by dieppa — APPROVED** [review]

**Pull request merged**